src/customFunctions/numberWithCommas.ts

```diff
@@ -0,0 +1,3 @@
+export function numberWithCommas(x: number) {
```

Comment:
as mentioned earlier, we have a function like this in formatting.ts

Reply:
Yeah forgot to delete the component but I have used the one from formatting.ts file wherever needed.